src/frontend/src/icp-eth/derived/cketh.derived.ts

```diff
@@ -117,3 +118,20 @@ export const ckErc20HelperContractAddress: Readable<OptionEthAddress> = derived(
 	([$ckEthMinterInfoStore, $ethereumTokenId]) =>
 		toCkErc20HelperContractAddress($ckEthMinterInfoStore?.[$ethereumTokenId])
 );
+
+/**
+ * The minter address.
+ */
+const ckMinterAddress: Readable<OptionEthAddress> = derived(
```

Comment:
I'm not convinced we should introduce a new global store for this scoped feature. On one hand, our idea is to deprecate tokenWillFallback; on the other, adding more global stores can impact performance. So unless this is truly used globally and not just in one or two functions, I don't think it should be a global store. WDYT?

Reply:
It is not using tokenWillFallback (or a derived), I think. But in any case, I understand what you mean with not adding more stores. Please check last commit, I moved the store logic inside the component.